### PR TITLE
Update Pidcat to 2.0.0

### PR DESCRIPTION
**Pull request by JakeWharton**
Changes: https://github.com/JakeWharton/pidcat/blob/master/CHANGELOG.md#version-200-2015-05-25

### CODE REVIEW

File: Library/Formula/pidcat.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class Pidcat < Formula
   homepage "https://github.com/JakeWharton/pidcat"
   head "https://github.com/JakeWharton/pidcat.git"
-  url "https://github.com/JakeWharton/pidcat/archive/1.4.1.tar.gz"
-  sha1 "89f806ae1fa3375ce188851c8c95fc1097467b82"
+  url "https://github.com/JakeWharton/pidcat/archive/2.0.0.tar.gz"
+  sha1 "7c33f7653894eccc9296a16f71fcf61cdee56cde"
```
Comment (DomT4) [inline Library/Formula/pidcat.rb]: If you wanted to bump this to `sha256`, :tada:, but it's not mandatory for existing formulae.

Comment (JakeWharton) [inline Library/Formula/pidcat.rb]: Done! :birthday:

Comment (DomT4) [inline Library/Formula/pidcat.rb]: You rock, Thanks!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/373db86592832c8f5bbb7cc0eec6ca73b98010e6. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:.

**Pull request closed without merging**